pdf.js

```diff
@@ -495,6 +518,7 @@ PDFViewer.prototype = {
       return;
     if (this.loadCallback_)
       this.loadCallback_(this.loadState_ == LoadState.SUCCESS);
+    window.dispatchEvent(new Event('pdf-loaded'))
```

Comment:
In the specs, we are using the page-title-updated event, which is not really a solid way of dealing with the loading I think, but I have found that at dom-ready the viewer is not initialized yet, that's why I left this event in there. This way, we don't have to wait for some extra event to happen on the webContents after load, we can just listen to this on the window object any time. I'd argue that this makes things easier, and is a one line change with not so much of maintenance impact.